cookiecutter.json

```diff
@@ -1,7 +1,14 @@
 {
     "full_name": "Your name",
     "email": "Your email address (eq. you@nelen-schuurmans.nl)",
+    "github_name": "Your github username",
     "project_slug": "your-project-with-dashes",
     "package_name": "{{ cookiecutter.project_slug.lower().replace('-', '_') }}",
+    "site_name": "{{ project_slug}}.lizard.net"
     "project_short_description": "A short description of the project"
```

Comment:
Github geeft hier en op de volgende regel de `:` rood weer. Is die weggehaald? Of is het een raar tekentje? Ik zie het even niet.